native_client/ds_git_version.sh

```diff
@@ -4,19 +4,32 @@ if [ `uname` = "Darwin" ]; then
    export PATH="/Users/build-user/TaskCluster/Workdir/tasks/tc-workdir/homebrew/opt/coreutils/libexec/gnubin:${PATH}"
 fi
 
-GIT_DIR="$(realpath "$(dirname "$(realpath "$0")")/../.git/")"
-if [ ! -d "${GIT_DIR}" ]; then
+DS_GIT_DIR="$(realpath "$(dirname "$(realpath "$0")")/../.git/")"
+if [ ! -d "${DS_GIT_DIR}" ]; then
    return 1
 fi;
 
-GIT_VERSION=$(git --git-dir="${GIT_DIR}" describe --long --tags)
+TF_GIT_DIR="$(realpath $(pwd)/tensorflow/../.git/)"
```

Comment:
Is TFDIR not available here? It'd be nice if we could make this work for local builds too.